rhui_provision/tasks/mon_serv_install.yml

```diff
@@ -0,0 +1,12 @@
+- name: update machine to latest
```

Comment:
I think we should capitalize the first letter in task names (helps them stand out when reading Ansible output, IMO), and also fully express what a task is doing. 

In this case, I would name this task "Yum update packages to latest available"